server/routes/shared/endOfServiceReport/endOfServiceReportPresenter.ts

```diff
@@ -11,6 +11,8 @@ export default class EndOfServiceReportPresenter {
     private readonly serviceCategories: ServiceCategory[]
   ) {}
 
+  readonly hrefBackLink = `/service-provider/referrals/${this.endOfServiceReport.referralId}/progress`
```

Comment:
End of service report can be viewed by probation practitioner as well, this gives them a link back to a page they can't view